rome_app/templates/rome_templates/about.html

```diff
@@ -26,12 +27,9 @@
-    <br />
-    <p>These printed books, maps, and single sheet prints each provided a showcase for the skills of artists and authors who displayed their knowledge of antiquity through imaging the city's ruins, or their knowledge of engineering by minutely illustrating its most successful projects, or their mastery of the topography and the history of the entire city through maps and views.  These books, always expensive, have often been dissected and sold page by page, or else kept locked in rare book libraries because of their value and their beauty.  They are written in archaic versions of several languages, which has led modern readers to ignore the texts and more technical or archaeological information that were always meant to accompany the images of, for example, Giovanni Battista Piranesi. Our goal is to create a digital library, research and teaching site that reunites text and image, and provides a corresponding base of bibliographic information, historical documentation and critical interpretation that will reveal patterns of practical and intellectual collaboration in the early modern project of printing the image of early modern Rome. Many of the illustrated books printed in the Renaissance were called "Theaters", designed to display for a reader the whole panoply of any one subject with special attention to its variety and diversification within any given rubric.  <strong>The Theater that was Rome</strong> looks at the creation of early modern Rome as a theater populated by a cosmopolitan group of engineers, artists, architects, authors, printers and publishers, gardeners, pilgrims, clerics, aristocrats and intellectual cardinals who contributed to the flood of printed information that proceeded from early modern interest in the physical and mythological city.</p>
-
+    <h2>{{about_title}}</h2>
+    {{about_text|markdown}}
+   
     <h2>People</h2>
     <p>Evie Lincoln (History of Art and Architecture), Faculty Lead</p>
     <p>Brian Repetto (GS, History of Art and Architecture)</p>
```

Comment:
can we put these "People" lines in the DB markdown as well? Also, do you have markdown in your local database that you're testing with? If so, could you please email it to me and I'll put it in the test server DB when I deploy?